api/envoy/admin/v3/certs.proto

```diff
@@ -34,11 +34,22 @@ message Certificate {
   repeated CertificateDetails cert_chain = 2;
 }
 
-// [#next-free-field: 7]
+// [#next-free-field: 8]
 message CertificateDetails {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.admin.v2alpha.CertificateDetails";
 
+  message OcspDetails {
+    // Seconds until expiration of the certificate's OCSP response.
+    uint64 seconds_until_expiration = 1;
+
+    // Indicates the time from which the OCSP response is valid.
+    google.protobuf.Timestamp valid_from = 2;
```

Comment:
Continuing the thread with @zuercher on modeling details here, how come we need both expiration time, valid from and seconds until expiration? Would it not be sufficient to just have valid_from (timestamp) and a duration indicating validity?

I don't want to bikeshed, but would prefer we at least avoid redundant fields.

Reply:
I'll remove the seconds field, although I think it has some utility for someone querying the admin API via a shell script.